tests/unit/type/list.tcl

```diff
@@ -311,7 +311,7 @@ start_server [list overrides [list save ""] ] {
         assert_equal [read_big_bulk {r rpop lst}] $str_length
         assert {[r llen lst] == 1}
         assert_equal [read_big_bulk {r rpop lst}] $str_length
-   } {} {large-memory}
+   } {} {largemem:skip}
```

Comment:
If you rename the tag you need to modify bitops.tcl too. 
But actually, I think the is name was better. It indicates the test requires large memory. 
Otherwise, if you refer CLI option, it should be `no-largemem:skip`. 
I think the should have been negated (hence my other comments). 
So bottom line, I suggest applying my comments, and then rename all the references to tag back to the old name.